tools/vagrant/README.md

```diff
@@ -116,7 +116,11 @@ deployment and run the "echo" action again to test. The following commands
 assume that you have `wsk` setup correctly in your PATH.
 ```
 # Set your OpenWhisk Authorization Key.
+
+# Non-Windows systems:
 wsk property set --apihost 192.168.33.16 --auth `vagrant ssh -- cat openwhisk/ansible/files/auth.guest`
+# Windows systems (Powershell):
+wsk property set --apihost 192.168.33.16 --auth $(vagrant ssh -- cat openwhisk/ansible/files/auth.guest)
```

Comment:
Using $(....) should work on Linux and Mac, have the same command for all

Reply:
OK, though changed the wording to imply that Powershell should be used when the deployment is hosted in Windows (as the default cmd prompt won't correctly substitute the result of expressions).